account_multicurrency_revaluation/i18n/fr.po

```diff
@@ -14,3 +15,455 @@ msgstr ""
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
+#. module: account_multicurrency_revaluation
+#: field:wizard.currency.revaluation,revaluation_date:0
+msgid "Revaluation Date"
+msgstr "Date de réévaluation"
+
+#. module: account_multicurrency_revaluation
+#: report:addons/account_multicurrency_revaluation/report/templates/unrealized_currency_gain_loss.mako:56
+msgid "TOTAL"
+msgstr "TOTAL"
+
+#. module: account_multicurrency_revaluation
+#: model:ir.model,name:account_multicurrency_revaluation.model_unrealized_report_printer
+msgid "unrealized.report.printer"
+msgstr "Détails.ecarts.changes"
+
+#. module: account_multicurrency_revaluation
+#: help:wizard.currency.revaluation,label:0
+msgid "This label will be inserted in entries description. You can use %(account)s, %(currency)s and %(rate)s keywords."
+msgstr "Ce champs corresponds a la desription des écritures. You pouvez utiliser les mots clés suivants: %(account)s, %(currency)s et %(rate)s."
```

Comment:
s/corresponds a la desription/correspond à la description/ (3 typos)
s/You/Vous